src/app/embed/adapters/merge.adapter.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { ReplaySubject } from 'rxjs';
+import { QSDAdapter } from './qsd.adapter'
+import { FeedAdapter } from './feed.adapter'
+import { AdapterProperties, hasMinimumParams, DataAdapter, getMergedValues } from './adapter.properties'
+
+const NO_EMIT_YET = Symbol();
+
+@Injectable()
+export class MergeAdapter {
+  private adapters: DataAdapter[]
+  constructor(
+    private FeedAdapter: FeedAdapter,
+    private QSDAdapter:  QSDAdapter
+	) { 
+    this.adapters = [this.QSDAdapter, this.FeedAdapter]
```

Comment:
Does this mean that all embed players will also load their xml feed, even if i'm only specifying QSAdapter params?